schema/test/unit/mutations/create_project_revision_test.sql

```diff
@@ -177,28 +241,74 @@ select is_empty(
   'creating a new project revision should not create a form_change record for the project_contact'
 );
 
+
+
 select results_eq(
   $$
-    select new_form_data, json_schema_name from cif.form_change
+    select form_data_record_id, previous_form_change_id, new_form_data, json_schema_name from cif.form_change
     where form_data_table_name = 'reporting_requirement' and project_revision_id = 2
     order by json_schema_name
   $$,
   $$
   values
-    ('{
+    (3,5,'{
+      "projectId":1,
+      "adjustedEmissionsIntensityPerformance": 98,
+      "baselineEmissionIntensity": 324.25364,
+      "comments": "comments",
+      "dateSentToCsnr": "2023-07-01T00:00:00-07:00",
+      "emissionFunctionalUnit":"tCO2e",
+      "measurementPeriodEndDate":"2023-07-01T00:00:00-07:00",
+      "measurementPeriodStartDate": "2023-06-01T00:00:00-07:00",
+      "postProjectEmissionIntensity": 124.35,
+      "productionFunctionalUnit": "unit",
+      "reportDueDate": "2023-06-15T00:00:00-07:00",
+      "reportType": "TEIMP",
+      "reportingRequirementIndex": 1,
+      "submittedDate": "2023-06-30T00:00:00-07:00",
+      "targetEmissionIntensity": 23.2357,
+      "totalLifetimeEmissionReduction": 44.4224
+      }'::jsonb, 'emission_intensity'::varchar),
+    (1,3,'{
       "projectId":1,
       "reportType": "General Milestone",
       "reportingRequirementIndex": 1
       }'::jsonb, 'milestone'::varchar),
-    ('{
+    (4,6,'{
```

Comment:
If the commit handler returns null, then the `form_data_record_id` is also null. So when `create_project_revision` makes a copy of the existing forms, it just copies the original one over and over again instead of getting the latest one